Resources data is throwing and exception instead of handling it

Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

## Describe the bug (Tell us what happens instead of the expected behavior) :
student_dashboard  | [2019-08-07 15:41:06 -0400] [23] [ERROR] Internal Server Error: /api/v1/courses/295532/resource_access_within_week
student_dashboard  | Traceback (most recent call last):
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
student_dashboard  |     response = get_response(request)
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
student_dashboard  |     response = self._get_response(request)
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
student_dashboard  |     response = self.process_exception_by_middleware(e, request)
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
student_dashboard  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
student_dashboard  |     return view_func(request, *args, **kwargs)
student_dashboard  |   File "/usr/local/lib/python3.6/site-packages/rules/contrib/views.py", line 141, in _wrapped_view
student_dashboard  |     return view_func(request, *args, **kwargs)
student_dashboard  |   File "/code/dashboard/views.py", line 291, in resource_access_within_week
student_dashboard  |     output_df['resource_id_part'], output_df['resource_name_part'] = output_df['resource_id_name'].str.split(';', 1).str
student_dashboard  | ValueError: not enough values to unpack (expected 2, got 0)
student_dashboard  | [2019-08-07 15:41:06 -0400] [23] [ERROR] "GET /api/v1/courses/295532/resource_access_within_week?week_num_start=12&week_num_end=14&grade=80-89&resource_type=Files HTTP/1.1" 500 24748


## Steps to Reproduce :
1. 
2.
3.

## Screenshots/Video (If possible, add media to help explain your problem) :

## Additional context (Add any other context about the problem here) :

--- a/dashboard/views.py
+++ b/dashboard/views.py
@@ -179,7 +179,7 @@ def resource_access_within_week(request, course_id=0):
     total_number_student = total_number_student_df.iloc[0,0]
     logger.info(f"course_id {course_id} total student={total_number_student}")
     if total_number_student == 0:
-        logger.info(f"For grade {grade} request there are no students in the percent grade range for course {course_id}")
+        logger.info(f"There are no students in the percent grade range {grade} for course {course_id}")
         return HttpResponse("{}")
 
     term_date_start = AcademicTerms.objects.course_date_start(course_id)

--- a/dashboard/views.py
+++ b/dashboard/views.py
@@ -178,6 +178,9 @@ def resource_access_within_week(request, course_id=0):
     total_number_student_df = pd.read_sql(total_number_student_sql, conn, params={"course_id": course_id})
     total_number_student = total_number_student_df.iloc[0,0]
     logger.info(f"course_id {course_id} total student={total_number_student}")
+    if total_number_student == 0:
+        logger.info(f"For grade {grade} request there are no students in the percent grade range for course {course_id}")
+        return HttpResponse("{}")
 
     term_date_start = AcademicTerms.objects.course_date_start(course_id)
 

